plugins/view-resources/src/utils.ts

```diff
@@ -63,6 +63,7 @@ import { get, writable } from 'svelte/store'
 import plugin from './plugin'
 import { noCategory } from './viewOptions'
 
+export { default as debounce } from 'lodash.debounce'
```

Comment:
I'm not happy with this.

Reply:
I'm not sure about writing custom implementation of exactly the same functionality. We also have this dependency in the "Collaborator" pod.

Reply:
But it is not used.

Let's switch to just a 
```js
clearTimeout(timeout); timeout = setTimeout(refresh, 200)
```

and it will be enough for now.

Reply:
Done.